apps/sr-frontend/src/app/App.tsx

```diff
@@ -30,6 +31,7 @@ export default function App() {
                 <Character />
                 <RelicEditor />
                 <Optimize />
+                <CharacterInventory />
```

Comment:
Put this in an Accordion please

Reply:
See https://github.com/frzyc/genshin-optimizer/pull/1881/commits/3a3a79cfa8aecc6d8a03fffe8ef070ab99b731a3

Reply:
Done.